stdlib/websub/src/main/ballerina/src/websub/hub_configuration.bal

```diff
@@ -78,3 +85,27 @@ function getSignatureMethod(SignatureMethod? signatureMethod) returns string {
 function getRemotePublishConfig(RemotePublishConfig? remotePublish) returns RemotePublishConfig {
     return remotePublish is RemotePublishConfig ? remotePublish : {};
 }
+
+function defaultOnMessageFunction(WebSubContent content) {
+    log:printDebug("Message received at the hub");
+}
+
+function defaultOnDeliveryFunction(string callback, string topic, WebSubContent content) {
+    log:printDebug("Content delivery to callback[" + callback + "] successful for topic[" + topic + "]");
```

Comment:
Let's use the function pointer approach instead, because there's concatenations. https://ballerina.io/learn/by-example/log-api.html

```suggestion
    log:printDebug(() => "Content delivery to callback[" + callback + "] successful for topic[" + topic + "]");
```